src/Footer.jsx

```diff
@@ -0,0 +1,50 @@
+const Footer = ({
+	pageNumber,
+	setPageNumber,
+	numArticles,
+	articlesPerPage,
+}) => {
+	const totalPages = Math.ceil(numArticles / articlesPerPage);
+	const pageNumbers = [<span key="0">Page: </span>];
+
+	const changePage = (event) => {
+		setPageNumber(Number(event.target.innerText));
```

Comment:
Try to avoid accessing the `innerText` property of event targets - you can pass the desired number as an argument

Reply:
`() => changePage(n)`